app/src/androidTest/java/org/kiwix/kiwixmobile/page/bookmarks/BookmarksRobot.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Kiwix Android
+ * Copyright (c) 2020 Kiwix <android.kiwix.org>
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package org.kiwix.kiwixmobile.page.bookmarks
+
+import applyWithViewHierarchyPrinting
+import com.schibsted.spain.barista.assertion.BaristaVisibilityAssertions.assertDisplayed
+import org.kiwix.kiwixmobile.BaseRobot
+import org.kiwix.kiwixmobile.Findable.StringId.ContentDesc
+import org.kiwix.kiwixmobile.Findable.Text
+import org.kiwix.kiwixmobile.R
+import org.kiwix.kiwixmobile.testutils.TestUtils.getResourceString
+
+/**
+ * Authored by Ayush Shrivastava on 29/10/20
+ */
+
+fun bookmarks(func: BookmarksRobot.() -> Unit) =
+  BookmarksRobot().applyWithViewHierarchyPrinting(func)
+
+class BookmarksRobot : BaseRobot() {
+  /** Pushed back robot rules due to lack of info to assert the correct screen */
+  init {
+    assertDisplayed(R.string.bookmarks_from_current_book)
+    // isVisible(Findable.StringId.TextId(R.string.bookmarks_from_current_book))
```

Comment:
What is happening here?

Reply:
It is to show that the `assertDisplayed()` from barista works and not the `isVisible()` one

Reply:
Very strange, eh well we can leave this one